File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int[] result = new int[2];

        for(int i = 0; i < numbers.length-1; i++){
            for(int j = numbers.length-1; j > 0; j--){
                if(numbers[i] + numbers[j] == target && i != j){
                    result[0] = i+1;
                    result[1] = j+1;
                    return result;
                }
            }
        }
        return result;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
from collections import Counter

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        freq = {}
        for i, x in enumerate(nums):
            comp = target - x
            if comp in freq:
                return [freq[comp], i]
            freq[x] = i